selfservice/hook/web_hook.go

```diff
@@ -389,9 +404,19 @@ func (e *WebHook) ExecuteSettingsPrePersistHook(_ http.ResponseWriter, req *http
 }
 
 func (e *WebHook) execute(ctx context.Context, data *templateContext) error {
-	// TODO fandom - fix webhooks
+	conf, err := newHttpConfig(e.conf)
+	if err != nil {
+		return fmt.Errorf("failed to parse http config: %w", err)
+	}
 	var (
-		httpClient     = e.deps.HTTPClient(ctx)
+		httpClient = e.deps.NamedHTTPClient(
+			ctx,
+			data.HookType+conf.sum,
+			httpx.ResilientClientWithMaxRetry(conf.retries),
+			httpx.ResilientClientWithConnectionTimeout(conf.timeout),
+			httpx.ResilientClientWithMinxRetryWait(conf.minWait),
+			httpx.ResilientClientWithMaxRetryWait(conf.maxWait),
+		)
```

Comment:
Maybe we could push this change to upstream

Reply:
I think we should add ACs to all such tickets